index.html

```diff
@@ -1,9 +1,106 @@
 <!DOCTYPE html>
 <html>
+<meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
 <head>
-	<title></title>
+<title>Cultural Club</title>
+<style type="text/css">
+	.hit-the-floor {
+color: #fff;
+font-size: 6em;
+font-weight: bold;
+font-family: Helvetica;
+text-shadow: 0 1px 0 #ccc, 0 2px 0 #c9c9c9, 0 3px 0 #bbb, 0 4px 0 #b9b9b9, 0 5px 0 #aaa, 0 6px 1px rgba(0,0,0,.1), 0 0 5px rgba(0,0,0,.1), 0 1px 3px rgba(0,0,0,.3), 0 3px 5px rgba(0,0,0,.2), 0 5px 10px rgba(0,0,0,.25), 0 10px 10px rgba(0,0,0,.2), 0 20px 20px rgba(0,0,0,.15);
+}
+
+.hit-the-floor {
+  text-align: center;
+}
+#img1
+{
+    float: left;
+}
+#img2
+{
+    float: right;
+}
+
+</style>
 </head>
 <body>
-<h1>Indian Institute of Information Technology</h1>
+<div class="hit-the-floor">
+	<div id="img1"><img src="a.jpg" width=250px height=150px></div>
```

Comment:
Stay consistent with css styling. Try making a CSS class and use it wherever possible.